ward/_config.py

```diff
@@ -26,8 +26,9 @@ def read_config_toml(project_root: Path, config_file: str) -> _ConfigDict:
         return {}
 
     try:
-        pyproject_toml = toml.load(str(path))
-    except (toml.TomlDecodeError, OSError) as e:
+        with open(path, encoding="utf-8") as f:
+            pyproject_toml = tomli.load(f)
```

Comment:
An alternative one-liner:
```suggestion
        pyproject_toml = tomli.loads(path.read_text(encoding="utf-8"))
```